packages/nocodb/src/models/View.ts

```
@@ -1596,6 +1613,42 @@ export default class View implements ViewType {
     );
   }
 
+  static async getSharedViewPath(
+    context: NcContext,
+    viewId,
+    ncMeta = Noco.ncMeta,
+  ) {
+    const view = await this.get(context, viewId, ncMeta);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)